src/main/kotlin/it/pagopa/touchpoint/jwtissuerservice/services/ReactiveAzureKVSecurityKeysService.kt

```diff
@@ -37,7 +38,7 @@ class ReactiveAzureKVSecurityKeysService(
     fun getCerts(): Flux<KeyVaultCertificate> {
         return certClient
             .listPropertiesOfCertificateVersions(azureSecretConfig.name)
-            .filter { it.isEnabled }
+            .filter { it.isEnabled && OffsetDateTime.now().isBefore(it.expiresOn) }
```

Comment:
```suggestion
            .filter { it.isEnabled && it.expiresOn > OffsetDateTime.now() }
```
🚲 I would use the mathematic syntax here for the comparison, found it a bit clearer

Reply:
I think natural language approach with method chaining is better suited for readability